src/components/ui/Shell/Shell.tsx

```diff
@@ -0,0 +1,273 @@
+          {name}
+        </h2>
+        <div className="md:col-span-3">
+          <ul role="list" className="space-y-16">
+            {cards.map((item, index) => {
+              return (
+                <UIMockupCard
+                  key={index}
+                  title={item.title}
+                  description={item.description}
+                  href={item.href}
+                  intent={intent}
+                />
+              );
+            })}
+          </ul>
+        </div>
+      </div>
+    </section>
+  );
+};
+const UIMockupCard = ({ ...props }: UIMockupCardProps) => {
+  const { title, description, intent, href } = props;
+  return (
+    <article className="flex flex-row md:items-baseline md:gap-8">
+      <div className="max-w-xl" />
+      <a className="mt-2 ">
+        <div className="group relative flex flex-col items-start md:col-span-3">
+          <h2
+            className={`text-base font-semibold tracking-tight ${textStyles({
+              intent,
+            })}`}
+          >
+            <div className={hoverStyles({ intent })}></div>
+            <span className="absolute -inset-y-6 -inset-x-4 z-20 sm:-inset-x-6 sm:rounded-2xl"></span>
+            <span className="relative z-10">{title}</span>
+          </h2>
+          <div
+            className={`relative z-10 mt-2 text-sm ${textCardStyles({
+              intent,
+            })}`}
+          >
+            <ReactMarkdown remarkPlugins={[remarkGfm]}>
+              {description}
+            </ReactMarkdown>
+          </div>
+
+          <div
+            aria-hidden="true"
+            className={`relative z-10 mt-4 flex items-center text-sm font-medium ${hrefStyles(
+              { intent }
+            )}`}
+          >
+            Check it out
+            <svg
+              viewBox="0 0 16 16"
+              fill="none"
+              aria-hidden="true"
+              className="ml-1 h-4 w-4 stroke-current"
+            >
+              <path
+                d="M6.75 5.75 9.25 8l-2.5 2.25"
+                strokeWidth="1.5"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              ></path>
+            </svg>
+          </div>
+        </div>
+      </a>
+    </article>
+  );
+};
+
+export const Shell = ({ intent, ...props }: ShellProps) => {
+  return (
+    <div>
+      <div className={shellStylesOuter({ intent })} />
+      <div className="relative">
+        <main className={shellStylesInner({ intent })}>
+          <div className="mx-auto max-w-2xl lg:max-w-5xl">
+            <header className="max-w-2xl">
+              <h1
+                className={`text-4xl font-bold tracking-tight ${textStyles({
+                  intent,
+                })} sm:text-5xl`}
+              >
+                About Me
+              </h1>
+              <div className={subHeaderStyles({ intent })}>
+                {"Some things about me."}
+              </div>
+            </header>
+            <UISection
```

Comment:
Let's have this component take a `sections` prop and then map those here

Reply:
I was a bit vague, something like 
```
sections.map(section => { ...section stuff, cards: section.entries.map(entry => {...card stuff})})
```

Reply:
Yea I pass an array of card objects to each `UISection` component and then map that array as individual `UIMockupCard` components. I think we are in agreement